RxCocoa/Common/RxCocoa.swift

```diff
@@ -287,6 +287,15 @@ func castOrFatalError<T>(value: AnyObject!) -> T {
     return result
 }
 
+func castOrFatalError<T>(value: Any!) -> T {
+    let maybeResult: T? = value as? T
+    guard let result = maybeResult else {
+        rxFatalError("Failure converting from \(value) to \(T.self)")
+    }
+    
+    return result
+}
+
```

Comment:
Is there any difference then just changing `AnyObject` parameter in `func castOrFatalError<T>(value: AnyObject!) -> T {`

to `Any!`?